crates/biome_cli/src/commands/check.rs

```diff
@@ -74,13 +77,35 @@ pub(crate) fn check(
         session.app.console,
         cli_options.verbose,
     )?;
+    let (editorconfig, editorconfig_diagnostics) = {
+        let search_path = loaded_configuration
+            .directory_path
+            .clone()
+            .unwrap_or_else(|| {
+                session
+                    .app
+                    .fs
+                    .working_directory()
+                    .unwrap_or_else(|| PathBuf::from("."))
```

Comment:
Instead of `PathBuf::from(".")`, I think we should use [`env::current_dir()`](https://doc.rust-lang.org/std/env/fn.current_dir.html) because that's what we use in other places. But I doubt it if this code will be ever reached.

Reply:
Yeah I didn't really know what to do here, `current_dir` is better.

Reply:
I just tried this and it turns out this code path is taken, pretty much only by tests. I've reverted the suggested change to use `current_dir` for now and just `unwrap_or_default` instead.

Reply:
Yes as @ematipico suggested, `unwrap_or_default` is the way to go.